pkg/app/src/components/Account/modules/CampaignsSection/CreatedCampaignsSection/loadingCampaignCard.tsx

```diff
@@ -1,39 +1,21 @@
-import React, { FC } from 'react'
+import React from 'react'
 
-import { Box, Card, CardContent, Container, Skeleton } from '@mui/material'
-import { useTheme } from '@mui/material/styles'
+import { Box, Card, CardContent, Skeleton } from '@mui/material'
 
-const LoadingCampaignCard: FC = () => {
-	const theme = useTheme()
+const LoadingCampaignCard = () => {
 	return (
-		<Card sx={{ minHeight: 388 }}>
+		<Card sx={{ maxHeight: 433 }}>
```

Comment:
where is this number coming from?

Reply:
this number is as the campaign card maxHeight so that the loading skeleton is the same

Reply:
yes, I can read that. usually fixed numbers should be dividable by eight,  not sure if we need fixed numbers there? @DarkNebula0